sample/src_polymer/manifest.json

```diff
@@ -1,12 +1,12 @@
-{
-  "name": "Crosswalk RealSense Sample App - Polymer",
-  "start_url": "index.html",
-  "xwalk_package_id": "org.crosswalk_project.rs_sample",
-  "xwalk_command_line": "--use-rs-video-capture",
-  "xwalk_windows_vendor": "Crosswalk Project",
-  "xwalk_extensions": [
-    "realsense_extensions/enhanced_photography",
-    "realsense_extensions/face",
-    "realsense_extensions/scene_perception"]
-}
-
+{
+  "name": "Crosswalk RealSense Sample App - Polymer",
+  "start_url": "index.html",
+  "xwalk_package_id": "org.crosswalk_project.rs_sample",
```

Comment:
Would this make a name conflict with old sample? Or we add polymer into it "org.corsswalk_project.rs_sample_polymer"?